### ISSUE TITLE
Sawmill worker auto detect

### ISSUE THREAD

**Issue by patjouk** [MozillaFoundation/mofo-devops#596]
To prevent bug [like this one](https://github.com/mozilla/network-pulse-api/pull/376) to happen, sawmill should auto-detect workers.

This should be done after #595

**Comment by cadecairos** [comment]
closing in favour of documentation: #613
